src/xbps/repositories/official/index.md

```diff
@@ -3,11 +3,23 @@
 Void provides other official repositories, which are maintained by the Void
 project, but not installed in the default configuration:
 
-- [debug](./debug.md): contains debugging symbols for packages
-- [multilib](./multilib.md): contains 32-bit libraries for 64-bit systems (glibc
-   only)
-- [multilib/nonfree](./multilib.md): contains non-free multilib packages
-- [nonfree](./nonfree.md): contains software packages with non-free licenses
-
-The next sections provide information on installing repositories, and summaries
-of the different repositories offered.
+| Package			| Description						| Link					|
+|-------------------------------|-------------------------------------------------------|---------------------------------------|
+| `void-repo-multilib`		| 32-bit compatibility packages 			| [multilib](./multilib.md)		|
+| `void-repo-mulitlib-nonfree`	| 32-bit compatibility packages with nonfree components | [multilib-nonfree](./multilib.md)	|
+| `void-repo-nonfree`		| packages with unfree licenses 			| [nonfree](./nonfree.md)			|
```

Comment:
```suggestion
| `void-repo-nonfree`		| packages with non-free licenses 			| [nonfree](./nonfree.md)			|
```

Also, definitely run this through the formatter. Inline tab characters will inevitably break text layout if you're using the wrong tab width.

Reply:
I didn't notice this. Definitely change this.